base/special/exp.jl

```diff
@@ -0,0 +1,137 @@
+# Based on FreeBSD lib/msun/src/e_exp.c
+# ====================================================
+# Copyright (C) 2004 by Sun Microsystems, Inc. All rights reserved. Permission
```

Comment:
The Julia license file needs to have an exception here, and the script that adds license headers needs to add an exception for this file. It's derived code so should usually retain its original licensing